README.md

```diff
@@ -1 +1,10 @@
-# palindrome_check
\ No newline at end of file
+# Palindrome Check
+In this assignment, you'll design and implement one of the string manipulation functions that is commonly asked during interviews.
+Remember that a string is an array of characters. Algorithms that worked on restricted arrays will work on strings as well.
+
+## Exercise
+* Design and implement a method that checks if the input string is a palindrome. The method should return true if the input string is a palindrome, and return false if it is not a palindrome.
+**Note**: Palindrome is a word, phrase or sentence that reads the same backwards as it does forwards. e.g. "madam"
+* Share and explain the time and space complexities for your solution.
+* At minimum, your implementation should pass the basic tests.
+**Note**: Do not use Ruby provided functionality for `.reverse` and `.reverse!`. You may use `.length`.
```

Comment:
We should have an extra blank line above this so that the note goes into its own paragraph.